recipes/astra-toolbox/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+case `uname` in
+  Darwin*)
+    CC="gcc -stdlib=libstdc++"
```

Comment:
Please use `clang` and `libc++` on macos

Reply:
Deleted these lines. Not building for OSX anyways.